tests/app/dao/notification_dao/test_notification_dao.py

```diff
@@ -1158,18 +1160,23 @@ def test_dao_get_notifications_by_to_field_matches_partial_phone_numbers(
     sample_template,
     search_term,
 ):
+    assert sample_template.template_type == SMS_TYPE, "This should be the default value."
 
     notification_1 = create_notification(
         template=sample_template,
         to_field='+447700900100',
-        normalised_to='447700900100',
+        normalised_to='1447700900100',
```

Comment:
After recent changes to notification-utils, normalized numbers seem to start with the county code.